CAIPs/caip-171.md

```diff
@@ -0,0 +1,64 @@
+---
+caip: 171
+title: Session Identifiers
+author: Olaf Tomalka (@ritave)
+discussions-to: <URL>
+status: Draft
+type: Standard
+created: 2022-11-09
+---
+
+## Simple Summary
+
+This CAIP defines an common identifier for representing an open session with a
+wallet, including both in-browser session tokens and API-based connection IDs.
+
+## Motivation
+
+Currently, sessions with wallet clients are tracked differently across different
+architectures: in-browser MetaMask sandboxes instance-specific connections based
+on origin which are referred to by tokens in the browser, while WalletConnect
+uses a topic-based pub/sub protocol keyed to a unique topic string, etc.
+Aligning specific aspects of session state can be hard without shared
+assumptions about session boundaries and tracking. A minimal assumption is that
+sessions are tracked by identifiers, and that different actors can update the
+session accordingly.
+
+## Specification
+
+> Such sections are considered non-normative.
+
+### Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" written in
+uppercase in this document are to be interpreted as described in [RFC
+2119](https://www.ietf.org/rfc/rfc2119.txt)
+
+### Definition
+
+Whenever a CAIP uses a name `SessionIdentifier` and has this CAIP in the
+`required` front-matter property, it SHALL be interpreted as reference to this
+specification.
+
+> Notice that there are no code constraints on the token value except
+> that it be expressed as a string with few reserve characters, under 256
```

Comment:
Remove the size limit from a non-normative section and let's put it into specification below

Reply:
oops let's leave that in the separate PR so it doesn't block this one